acceptance/redis32/redis-ha.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+set -xue
+
+
+# get all pods by service_id
+curl -ks -u"${K8S_USERNAME}:${K8S_PASSWORD}" "${K8S_APISERVER}/api/v1/namespaces/default/pods/" | \
+jq '.items[] | select( .metadata.name | test( "'"${SERVICE_ID}"'" ) ) | { name: .metadata.name, node: .status.hostIP, ip: .status.podIP }'
```

Comment:
yeah I was even thinking it would be better to have some of these in the `main.go` in the acceptance tests as routes I can hit from the `redis-ha.sh`.